src/content/basics/api/index.md

```diff
@@ -0,0 +1,69 @@
+
+At Giant Swarm, when we say "control plane", we talk about the Kubernetes Cluster that runs all the operational and monitoring workloads which are needed to create and manage the "tenant clusters", which are the clusters you create to run your actual workloads. This is not to be confused with how the Kubernetes project uses the same term, simply meaning the master nodes of a Kubernetes cluster.
+
+The control plane is a Kubernetes cluster. Your tenant clusters and other associated resources are represented in that cluster as custom resources. To access these, you can use the Kubernetes API of the cluster that forms the control plane, or in short, the Control Plane Kubernetes API. For the sake of brevity, we'll abbreviate this as _CP-K8s-API_.
+
+### How to gain access
+
+To access the CP-K8s-API, Giant Swarm configures access to the API via OIDC. Please contact Solution Engineer (SE) to sort out the details.
+
+Currently we normally provide read-only access by default. As we are currently working on a fine-grained way to control permissions to resources, having write access right now means fully unrestricted access and should only be granted to select individuals.
+
+TODO: Explain how setting up write is different from read and what else it requires.
+
+### How to use
+
+We recommend that you become familiar with the API via `kubectl`, navigating your resources via the CP-K8s-API. Besides general Kubernetes know-how this will require only a bit of structural knowledge:
+
+#### How we organize resources in namespaces
+
+This is quickly explained: we create one namespace for each tenant cluster, where the namespace name is equal to the tenant cluster ID. All cluster specific resources reside in the namespace of that tenant cluster.
+
+### Which custom resources we use for what purpose
+
+For this, we provide some resources that should help you:
+
+- The guide [Creating tenant clusters via the Control Plane Kubernetes API](/guides/creating-clusters-via-crs-on-aws/) explains step by step how you can create a cluster and node pools via the CP-K8s-API. Here you learn about all the custom resources a cluster comprises.
+- The [App Platform](/basics/app-platform/) introduction outlines the several custom resources involved when managing app catalogs and apps.
+- Our [Control Plane Kubernetes API Reference](https://docs.giantswarm.io/reference/cp-k8s-api/) provides detailed documentation on all the custom resources we use with the various providers and their versions and schema.
+
+### Feedback is welcome
+
+We are keen to learn from you about your experience with accessing the CP-K8s-API, with navigating the custom resources via the API, with our reference documentation and the user guides we provide. This helps us provide more and better material and improve to make the journey more seamless, more satisfactory for you.
+
+So please, don't hesitate to give your feedback in your Slack channel.
+
+## The Giant Swarm Rest API (deprecated) {#rest-api}
```

Comment:
I'd still move this to the top for now. This should be the API that first users eg if we onboard customers should use.

Reply:
Hm. If this comes first the description also doesn't fit anymore.

Reply:
+1 for starting with this part, while making the necessary adjustments to the new content.

Reply:
Will do